sdk/mixedreality/ci.yml

```diff
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/mixedreality/
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - common/mixedreality/
```

Comment:
Should this be `sdk/mixedreality` as well?

Reply:
Yep. Probably so. Thanks for the catch.

Reply:
Fixed